templates/site/js/node.js

```diff
@@ -13,21 +13,15 @@ const converters = {
     },
     "status":(val) =>{
         let node_status = document.createElement("span")
-        const first_letter = val.charAt(0);
-        const first_letter_capitalize = first_letter.toUpperCase();
-        const remaining_word = val.slice(1);
-        const full_word = first_letter_capitalize + remaining_word
+        const full_word = capitalize(val);
         node_status.innerText = full_word;
         node_status.classList.add(val === "online" ? "node-online" : "node-offline")
         return node_status
     },
     "registration_status":(val) => {
         let registration_status_elem = document.getElementById("registration_status");
         registration_status_elem.classList.add("error-text");
-        const first_letter = val.charAt(0);
-        const first_letter_capitalize = first_letter.toUpperCase();
-        const remaining_word = val.slice(1);
-        const full_word = first_letter_capitalize + remaining_word
+        const full_word = capitalize(val);
```

Comment:
No need for a variable, just return it directly.

Reply:
Changed so it returns the function directly.